logbook-core/src/main/java/org/zalando/logbook/DefaultHttpLogWriter.java

```diff
@@ -81,7 +81,7 @@ public DefaultHttpLogWriter(final Logger logger, final Level level) {
         }
     }
 
-    @VisibleForTesting
+//    @VisibleForTesting
```

Comment:
I would suggest introduction of own annotation with same name/semantics or just removal instead of commenting out